src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -4570,6 +4575,17 @@ class CreateViewStatementSegment(ansi.CreateViewStatementSegment):
     )
 
 
+class ColumnQuoteSegment(BaseSegment):
+    """Double quotes column segment overriding default ColumnReferenceSegment."""
```

Comment:
Why not override `ColumnReferenceSegment`? Or can this alternative only be used in certain cases?